docker/run_celery.bash

```diff
@@ -3,11 +3,7 @@ set -e
 
 source /etc/profile
 
-CELERYD_TASK_SOFT_TIME_LIMIT="${CELERYD_TASK_SOFT_TIME_LIMIT:-$((15*60))}"
```

Comment:
@esmail, yes: https://github.com/kobotoolbox/kobocat/commit/b6e30094f2d69a8f471c13f61d2a47696d9c0a7f. Surprisingly (to me), the command-line limits [were overridden by `@task` decorators](https://www.flowdock.com/app/kobotoolbox/kobo/messages/969413), so I standardized everything to use environment variables (and the same time limits for all jobs). The defaults are 30 and 35 minutes for soft and hard time limits, respectively.